File: 백준/Bronze/1330. 두 수 비교하기/두 수 비교하기.go
package main

import (
	"bufio"
	"fmt"
	"os"
)

func main() {
	var reader *bufio.Reader = bufio.NewReader(os.Stdin)
	var writer *bufio.Writer = bufio.NewWriter(os.Stdout)
	defer writer.Flush()
	var a, b int
	str := []rune("")
	fmt.Fscan(reader, &a, &b)
	if a == b {
		str = []rune("==")
	} else if a > b {
		str = []rune(">")
	} else {
		str = []rune("<")
	}
	fmt.Fprintln(writer, string(str))
}

File: 백준/Bronze/1330. 두 수 비교하기/두 수 비교하기.rs
use std::io::{self, BufRead, BufReader, Write};

fn main() {
    let mut reader = BufReader::new(io::stdin().lock());
    io::stdout().flush().unwrap();

    let mut input = String::new();
    reader.read_line(&mut input).expect("");

    let mut numbers = input.trim().split_whitespace();
    let num1: i32 = numbers.next().expect("").parse().expect("");
    let num2: i32 = numbers.next().expect("").parse().expect("");

    let mut answer = "==";

    if num1 < num2 {
        answer = "<";
    } else if num1 > num2 {
        answer = ">";
    }

    println!("{answer}");
}
